lib/ChunkTemplate.js

```diff
@@ -12,6 +12,8 @@ function ChunkTemplate(outputOptions) {
 module.exports = ChunkTemplate;
 
 ChunkTemplate.prototype = Object.create(Template.prototype);
+ChunkTemplate.prototype.constructor = ChunkTemplate;
```

Comment:
As i run the test, it failed. but in fact i passed, because the failed ones were tests for some third-party plugin which couldn't find their dependencies. They don't connect to our topic.

So, it turns out, there is no problem here. Could you state more to replay the game?

Btw, Although I respect your comments, it's really noisy below and in my mailbox.